discojs/discojs-core/src/task/display_information.ts

```diff
@@ -14,7 +14,10 @@ export interface DisplayInformation {
   dataExample?: DataExample[]
   // TODO no need for undefined
   headers?: string[]
+  // Displays the image at this URL in the UI as an example when connecting data
   dataExampleImage?: string
+  // URL to download a dataset for the task, is displayed in the UI when asking to connect data
+  sampleDatasetLink?: string
```

Comment:
better to use an URL as to avoid issues with parsing it

```suggestion
  sampleDatasetLink?: URL
```